src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -364,9 +364,10 @@ public Future<InvoiceLine> createInvoiceLine(InvoiceLine invoiceLine) {
       .compose(v -> createInvoiceLine(ilProcessing, requestContext))
       .compose(v -> orderService.createInvoiceOrderRelation(ilProcessing.getInvoiceLine(), buildRequestContext())
         .recover(throwable -> {
+          var param = new Parameter().withKey("invoiceLineId").withValue(invoiceLine.getId());
           var errorParam = new Parameter().withKey("errorMessage").withValue(throwable.getMessage());
-          var error = ORDER_INVOICE_RELATION_CREATE_FAILED.toError().withParameters(List.of(errorParam));
-          logger.error("Failed to create invoice order relation", throwable);
+          var error = ORDER_INVOICE_RELATION_CREATE_FAILED.toError().withParameters(List.of(param, errorParam));
+          logger.error("Failed to create invoice '{}' order relation", invoiceLine.getId(), throwable);
```

Comment:
Failed to create invoice line '{}' order relation

Reply:
thanks, improved message